bundles/org.palladiosimulator.spd.semantic.transformations/transformations/spd/common/AdjustmentCalculator.qvto

```diff
@@ -32,6 +34,28 @@ helper Interprete_adjustmentType(adjustment: AdjustmentType, currentSize: Intege
 	return desiredChange;
 }
 
+/**
+* Interpretation of the GroupSizeConstraints given the currentSize of the TargetGroup and the desiredChange. 
+* The desiredChange is a computation of how many units to add or remove from the target group.
+* The interpretation below bounds the desiredChange so that it yields a number which does not exceed the specified groupSizeConstraints.
+*
+* @param currentSize			The number of elements in the target group.
+* @param desiredChange			The computed desired change given a certain type of adjustment.
+* @param groupSizeConstraints	The TargetGroupSizeConstraint in the model.
+* @return Real 			The modified desired change in capacity for the target group to respect bounds. 
+*/
+helper Interprete_GroupSizeConstraints(currentSize:Integer, desiredChange:Real, groupSizeConstraints:TargetGroupSizeConstraint) : Real {
+	assert fatal(groupSizeConstraints.minSize>=1) with log('A minimum below 1 is not permitted for a group size constraints.');
+	assert fatal(groupSizeConstraints.maxSize>1) with log('The maximum group size constraint equal or below 1 is not permitted for a group size constraint.');
```

Comment:
i'd recommend to also check that `maxSize > minSize`, which is currently allowed, but does not make much sense to me.  
or is that supposed to be checked in another place?